primer/brand
# Add Card component

## Summary

The first iteration of the Card component. The component now covers all the necessities for the Galaxy Phase 2 (composable API that allows a `Heading` and `Description`) and will be upgraded following the alignment work that is happening across .com pages (see more: https://github.com/github/brand-marketing-design/issues/720)

This closes: https://github.com/github/galaxy.github.com/issues/121
Tracking issue: https://github.com/github/primer/issues/1806
Figma branch: https://www.figma.com/file/BJ95AjraesmRCWsKA013GS/branch/sZ6jw7aenZ9VWl0dLM9vrK/Primer-Brand?node-id=5407%3A32782&t=iQ8GUOpbHoYhemEe-1

## List of notable changes:

- **added** `Card` component with `Heading` and `Description` items.
- **added** basic functional primitives.  
- **added** default story to showcase `Card`.
- **added** documentation.

## Steps to test:

- Open the CTAForm component in CI-deployed preview environment
- Go to Card story in Storybook
- Verify that the Card behaves as described in the following issue.

👀 Storybook: https://primer-f13cea2127-26139705.drafts.github.io/storybook/?path=/story/components-card--default
📖 Docs: https://primer-f13cea2127-26139705.drafts.github.io/components/Card

## Contributor checklist:


## Reviewer checklist:


## Screenshots:

> Card in light mode
> <img alt="Screenshot 2023-02-15 at 17 13 15" src="https://user-images.githubusercontent.com/912236/219086621-db91a339-a5d1-4821-ada7-685bd154ea3b.png">
> 

> Card in dark mode
> <img  alt="Screenshot 2023-02-15 at 17 13 19" src="https://user-images.githubusercontent.com/912236/219086632-f2edf0ad-6698-4c89-b6c1-5ceec9c1f554.png">
>
